components/centraldashboard/public/components/utilities-mixin_test.js

```diff
@@ -1,37 +1,117 @@
+/* eslint-disable max-len */
 import utilitiesMixin from './utilities-mixin';
+import {html} from '@polymer/polymer/polymer-element.js';
 
-class TestClass extends utilitiesMixin(Object) {
 
+class TestClass extends utilitiesMixin(Object) {
 }
 
+const PASS_FUNCTION = (_) => _;
+
 describe('Utilities Mixin', () => {
     let testObject;
 
     beforeEach(() => {
         testObject = new TestClass();
+        // Since base class is not HTML
+        testObject.dispatchEvent = PASS_FUNCTION;
```

Comment:
I would suggest using a mock here that way you can verify interactions easily with it.

Reply:
It, unfortunately, does not allow you to mock a field that never was defined. Since our test class is not really an HTML element, `dispatchEvent` is not a defined value on it. I could define this method within the class, and mock it here, however?

Reply:
That's not a problem. Just use [jasmine.createSpy()](https://jasmine.github.io/api/3.4/jasmine.html#.createSpy). Then you have a bare spy object that you can later. See my comment in the method where `dispatchEvent` is used below.

Reply:
Done!